pygfx/renderers/wgpu/_wgpurenderer.py

```diff
@@ -19,13 +17,13 @@
 # Definition uniform struct with standard info related to transforms,
 # provided to each shader as uniform at slot 0.
 # todo: a combined transform would be nice too, for performance
-stdinfo_uniform_type = Struct(
-    cam_transform=mat4,
-    cam_transform_inv=mat4,
-    projection_transform=mat4,
-    projection_transform_inv=mat4,
-    physical_size=vec2,
-    logical_size=vec2,
+stdinfo_uniform_type = dict(
+    cam_transform=("float32", (4, 4)),
+    cam_transform_inv=("float32", (4, 4)),
+    projection_transform=("float32", (4, 4)),
+    projection_transform_inv=("float32", (4, 4)),
+    physical_size=("float32", 2),
+    logical_size=("float32", 2),
```

Comment:
I can't say I think this is particularly pretty, but at least it's generic, and familiar to (some) numpy users.

We can make up any format we like to define structs, so if you have other ideas, speak up :)

Reply:
since you're passing it along to numpy untouched, does that mean you can also use e.g. `f4` instead of `float32`?

Reply:
Yes. That would make it a bit shorter.

Reply:
Well I don't mean to say you should, but I just wanted to confirm that you can :)